mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -1438,6 +1438,12 @@ func ResourceContainerCluster() *schema.Resource {
 								},
 							},
 						},
+						"resource_manager_tags": {
```

Comment:
Can you please add this new field to the resource documentation file?
mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

Reply:
added to node_pool_auto_config block